core/src/main/java/org/apache/iceberg/deletes/BitmapPositionDeleteIndex.java

```diff
@@ -42,4 +42,9 @@ public void delete(long posStart, long posEnd) {
   public boolean deleted(long position) {
```

Comment:
Is there a reason why we didn't use `isDeleted`? That is the normal convention for boolean methods.

Reply:
Made the change in the new commit.